components/Header.tsx

```diff
@@ -12,11 +12,11 @@ const tabs = [
 
 export const Header = () => {
   return (
-    <div className="flex flex-row w-full justify-between">
-      <div></div>
+    <header className="flex flex-row w-full items-center justify-between mx-auto">
+      <div className="font-bold flex-1">L2 Flex Voting</div>
```

Comment:
It seems like the navbar is slightly off center now. I thought the previous looked better, but we can document in another issue.

Reply:
re: slightly off center, it may *appear* that way because of the justification on either the wallet connect or the "bridge" heading, but 
on my browser it's center: here's a couple screenshots. curious to see how it looks on your end.

<img width="1140" alt="Screenshot 2023-10-13 at 11 04 46 AM" src="https://github.com/ScopeLift/l2-flexible-voting-frontend/assets/5423313/5e735520-7735-4bd8-af55-257a9661d3f7">

<img width="1719" alt="Screenshot 2023-10-13 at 11 04 34 AM" src="https://github.com/ScopeLift/l2-flexible-voting-frontend/assets/5423313/28d6fe1e-36d9-4a09-a808-ec988c3565dc">